docs/visualize/controls.asciidoc

```diff
@@ -0,0 +1,61 @@
+[[controls]]
+== Controls Visualization
+
+Controls provide the ability to add interactive inputs to Kibana Dashboards.
+Controls can be used to create two types of inputs; dropdowns and range sliders.
+
+image::images/controls/controls_in_dashboard.png[]
+
+=== Configuration
+Create Controls visualizations by following the <<createvis,Creating a Visualization>> instructions.
+Select *Controls* as the visualization type.
+
+==== Dropdown
+
+Dropdown inputs allow users to easily filter content by selecting options from a list.
+The dropdown options list is dynamically populated with the results of a terms aggregation.
+
+Add dropdown inputs to your Controls Visualization by selecting *Options list* and clicking *Add*.
+
+image::images/controls/dropdown_control_editor.png[]
+
+*Control Label*:: By default, the control label will be the *Field*. Set this configuration to customize the control's label.
+*Index Pattern*:: Specify the data set by selecting an <<index-patterns,Index Pattern>>.
+*Field*:: Select the Field that will be used to populate the options list
+and will be filtered on when users interact with the input.
+The list of available fields is derived from the selected *Index Pattern* configuration.
+*Parent control*:: Set parent control to chain dropdown inputs so that the selection of the first input
+will filter the terms of the second input. Configuration is only available when multiple dropdowns have been created.
+*Multiselect*:: When enabled, the dropdown will allow users to select multiple options.
+*Size*:: Use the size parameter to define the number of options.
```

Comment:
Maybe here add a comment that the "search-as-you-type" functionality is limited to the result set returned.

Use the size parameter to define the number of options.  Note if you have more unique values than this number, some fields will not be available to select as search is limited to the result set limited by this size parameter.  Useful for high cardinality fields to prevent too large of a result set being retrieved. 

Actually since both this doc and that PR are going into 6.4 maybe just hold off on this PR until that one is submitted? Then we can decide if size should be allowed with dynamic options off, and add more reasoning for why it's useful with high cardinality fields when dynamic options is on.

Reply:
How about the other way around? How about getting this PR in before the dynamic options PR so it can be back-ported to 6.3 cleanly and then adding the Dynamic Options documentation as part of PR https://github.com/elastic/kibana/pull/18985?